packages/client/src/Loc.ts

```diff
@@ -113,6 +113,8 @@ export interface MergedFile extends Partial<AddedOn>, Published {
     reviewedOn?: string;
     submitter: ValidAccountId;
     size: bigint;
+    acknowledgedByOwner: boolean;
+    acknowledgedByVerifiedIssuer: boolean;
```

Comment:
Why not expose the date-time (exposed by the backend) instead of a boolean (on-chain and backend)?

Reply:
Good suggestion: addressed by this issue: https://github.com/logion-network/logion-internal/issues/976